tools/expotools/src/commands/CreateUnimodule.ts

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import chalk from 'chalk';
+import JsonFile from '@expo/json-file';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+
+import { PACKAGES_DIR, EXPO_DIR } from '../Constants';
+
+type ActionOptions = {
+  name: string;
+};
+
+async function generateModuleWithExpoCLI(unimoduleDirectory) {
+  console.log(`Creating new unimodule under ${chalk.magenta(path.relative(EXPO_DIR, unimoduleDirectory))}...`);
+
+  await spawnAsync('expo', ['generate-module', unimoduleDirectory], {
+    cwd: EXPO_DIR,
+    stdio: 'inherit',
+  });
+}
+
+async function setupExpoModuleScripts(unimoduleDirectory) {
+  const packageJsonPath = path.join(unimoduleDirectory, 'package.json');
+  const packageJson = new JsonFile(packageJsonPath);
+  const moduleScriptsVersion = await JsonFile.getAsync(path.join(PACKAGES_DIR, 'expo-module-scripts', 'package.json'), 'version', '') as string;
+
+  console.log(`Installing ${chalk.bold.green('expo-module-scripts')}...`);
+
+  await spawnAsync('yarn', ['add', '--dev', `expo-module-scripts@~${moduleScriptsVersion}`], {
```

Comment:
There's no JS–native interface for expo-module-scripts and we version it like pure-JS modules:
```suggestion
  await spawnAsync('yarn', ['add', '--dev', `expo-module-scripts@^${moduleScriptsVersion}`], {
```